### DISCUSSION TITLE
How to make TOC at the right side?

### DISCUSSION THREAD

**Discussion by TerryLiang** [requarks/wiki#5959] [Help / Questions] [answered]
I have upgraded to the newest version 2.5.294.
How can I set the TOC to the right side?

**Comment by lovelock** [comment]
No, you can't if you do not hack the code. Check this out https://github.com/requarks/wiki/blob/d6d88ed9765e6a0748b2e13767b8f2388fde8054/client/themes/default/components/page.vue#L223

**Reply by NGPixel** [reply]
Incorrect, see my answer.

**Comment by NGPixel** [comment, marked as answer]
You can select which side from the Admin > Theme section.

**Reply by lovelock** [reply]
<img width="851" alt="image" src="https://user-images.githubusercontent.com/2208731/208627297-0bf0a11c-affb-4dfb-a5c9-f25a2beb794c.png">

my wiki is based on version 2.5.219, and it seems the option is not available for this version. I've checked the release notes and nothing related to this topic was mentioned.

**Reply by NGPixel** [reply]
Update to the latest version.
